app/repository/RaceRepositoryImpl.java

```diff
@@ -1,16 +1,22 @@
 package repository;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import models.Race;
 import play.db.jpa.JPAApi;
+import play.libs.Json;
 import repository.interfaces.RaceRepository;
 
 import javax.inject.Inject;
 import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.TypedQuery;
 import java.util.List;
 import java.util.concurrent.CompletionStage;
 import java.util.function.Function;
+import java.util.stream.Stream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.stream.Stream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)